CLI release v1.14.1 (#2169)
astronomer/docs: astro/cli/release-notes.md

This document provides a summary of all changes made to the [Astro CLI](cli/overview.md). For general product release notes, go to [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).

## Astro CLI 1.14.1

Release date: April 20, 2023

### Bug fixes

> - Fixed an issue where the command `astro workspace user list` would fail when being used with Workspace API Tokens.

Reviewer: ```suggestion
- Fixed an issue where `astro workspace user list` didn't work when using a Workspace API token.
```
